test/data/testinit.js

```diff
@@ -256,14 +256,11 @@ if ( !window.__karma__ ) {
 QUnit.isSwarm = ( QUnit.urlParams.swarmURL + "" ).indexOf( "http" ) === 0;
 QUnit.basicTests = ( QUnit.urlParams.module + "" ) === "basic";
 
-// Async test for module script type support
-function moduleTypeSupported() {
-	var script = document.createElement( "script" );
-	script.type = "module";
-	script.text = "QUnit.moduleTypeSupported = true";
-	document.head.appendChild( script ).parentNode.removeChild( script );
-}
-moduleTypeSupported();
+// Support: IE 11+
+// A variable to make it easier to skip specific tests in IE, mostly
+// testing integrations with newer Web features not supported by it.
+QUnit.isIE = /trident\/7\.0/i.test( window.navigator.userAgent );
+QUnit.testUnlessIE = QUnit.isIE ? "skip" : "test";
```

Comment:
Should we structure this as a function rather than a method name?
```suggestion
QUnit.testUnlessIE = QUnit.isIE ?
	QUnit.skip :
	QUnit.test;
```
```patch
-QUnit[ QUnit.testUnlessIE ]( "css(--customProperty)", function( assert ) {
+QUnit.testUnlessIE( "css(--customProperty)", function( assert ) {
```

Reply:
Good idea, that makes much more sense.

Reply:
PR updated.